docs/stochastic.rst

```diff
@@ -53,20 +53,38 @@ Solvers
     jaxopt.OptaxSolver
     jaxopt.PolyakSGD
 
-Example::
+Optax solvers
+~~~~~~~~~~~~~
+
+`Optax <https://optax.readthedocs.io>`_ solvers can be used in JAXopt using
+:class:`OptaxSolver <jaxopt.OptaxSolver>`. Here's an example with Adam::
+
+    from jaxopt import OptaxSolver
 
     opt = optax.adam(learning_rate)
     solver = OptaxSolver(opt=opt, fun=ridge_reg_objective, maxiter=1000)
 
+See `common optimizers
+<https://optax.readthedocs.io/en/latest/api.html#common-optimizers>`_ in the
+optax documentation for a list of available stochastic solvers.
+
+Adaptive solvers
+~~~~~~~~~~~~~~~~
+
+In addition, JAXopt supports :class:`PolyakSGD <jaxopt.PolyakSGD>`, a solver
+which computes step sizes in an adaptive manner using function values.
```

Comment:
adaptive might be misleading here. In the stochastic literature adaptive typically means diagonal rescaling (Adagrad-style). How about calling them parameter-free solvers?

Reply:
Which is not really true...

Reply:
I think at least it would be good to state what is meant here by adaptive.